subprojects/scala/src/main/java/org/gradle/api/tasks/ScalaRuntime.java

```diff
@@ -155,7 +155,7 @@ public void visitDependencies(TaskDependencyResolveContext context) {
      * @return a Scala Jar file with the specified appendix
      */
     @Nullable
-    public File findScalaJar(Iterable<File> classpath, String appendix) {
+    public static File findScalaJar(Iterable<File> classpath, String appendix) {
```

Comment:
This change is API breaking - any chance you can undo it?

Reply:
makes sense.... https://github.com/gradle/gradle/pull/18347/commits/d528e34be170aa3ae782bf272b97c66c909d40eb takes care of it